rps.js

```diff
@@ -0,0 +1,44 @@
+function playGame(userChoice){
+    pcChoice = Math.round(Math.random() * Math.round(3))
```

Comment:
As I see ```pcChoice``` is a random number start from 0 to 3 (0, 1, 2, 3) but we have 3 options so we need to enhance the range:
```Math.round(Math.random() * (max - min) + min);```